<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LecteurVue</class>
 <widget class="QMainWindow" name="LecteurVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lecteurvue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0,1,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="btnLancerDiapo">
        <property name="text">
         <string>Lancer Diaporama</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="titreDiapo">
        <property name="text">
         <string>Titre du diaporama</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnArreterDiapo">
        <property name="text">
         <string>Arrêter le diaporama</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="titreImage">
        <property name="text">
         <string>Titre de l'image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="catImage">
        <property name="text">
         <string>Catégorie</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnPrec">
        <property name="text">
         <string>Précédent</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerEntrePrecEtSuiv">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSuiv">
        <property name="text">
         <string>Suivant</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuParam_tres">
     <property name="title">
      <string>Paramètres</string>
     </property>
     <widget class="QMenu" name="menuMode">
      <property name="title">
       <string>Mode</string>
      </property>
      <addaction name="actionModeAuto"/>
      <addaction name="actionModeManuel"/>
     </widget>
     <addaction name="actionCharger_Diaporama"/>
     <addaction name="menuMode"/>
     <addaction name="actionChangerVitesseDefilement"/>
    </widget>
    <addaction name="menuParam_tres"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos_de"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionCharger_Diaporama">
   <property name="text">
    <string>Charger Diaporama</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos_de">
   <property name="text">
    <string>A propos de</string>
   </property>
  </action>
  <action name="actionX0_5">
   <property name="text">
    <string>x0.5</string>
   </property>
  </action>
  <action name="actionX0_75">
   <property name="text">
    <string>x0.75</string>
   </property>
  </action>
  <action name="actionX1">
   <property name="text">
    <string>X1</string>
   </property>
  </action>
  <action name="actionX1_25">
   <property name="text">
    <string>X1.25</string>
   </property>
  </action>
  <action name="actionX1_5">
   <property name="text">
    <string>X1.5</string>
   </property>
  </action>
  <action name="actionX1_75">
   <property name="text">
    <string>X1.75</string>
   </property>
  </action>
  <action name="actionX2">
   <property name="text">
    <string>X2</string>
   </property>
  </action>
  <action name="actionModeAuto">
   <property name="text">
    <string>Automatique</string>
   </property>
  </action>
  <action name="actionModeManuel">
   <property name="text">
    <string>Manuel</string>
   </property>
  </action>
  <action name="actionChangerVitesseDefilement">
   <property name="text">
    <string>Changer la vitesse de défilement du diaporama</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
